src/Google.Api.Gax.Grpc/ApiCallRetryExtensions.cs

```diff
@@ -86,7 +86,7 @@ internal static class ApiCallRetryExtensions
                         {
                             throw;
                         }
-                        scheduler.Sleep(actualDelay);
+                        scheduler.Delay(actualDelay, callSettings.CancellationToken.GetValueOrDefault()).Wait();
```

Comment:
Doh - thought I'd got all of them, but no... because I wasn't calling WaitAndUnwrapExceptions, which I should have been :) Will fix.